src/test_lib_json/main.cpp

```diff
@@ -350,7 +350,7 @@ JSONTEST_FIXTURE_LOCAL(ValueTest, arrayInsertAtRandomIndex) {
   JSONTEST_ASSERT_EQUAL(Json::Value("index1"), array[2]);
   JSONTEST_ASSERT_EQUAL(Json::Value("index2"), array[3]);
   // checking address
-  for (int i = 0; i < 3; i++) {
+  for (Json::ArrayIndex i = 0; i < 3; i++) {
```

Comment:
This is fine, but I'm kind of curious why this error just started happening. Any idea?

Reply:
New apple clang compiler with Mac 10.15, XCode 11.2 refused to compile the previous code due to ambiguities.